From 9591f71d7c1b3b9386e5499d7c9675cd6b7674ab Mon Sep 17 00:00:00 2001
From: Mohsen Dolaty <mohsen.dolaty@amd.com>
Date: Mon, 15 Nov 2021 16:38:41 -0600
Subject: [PATCH 1/1] ARM:dts:aspeed Remove board SPD for sp5 platforms till
 i3c is enabled

Removed board SPD for SP5 till i3c is enabled by adding the flag
OPENBMC_I3C_ENABLE  Also corrected i2c-1 bus physical address from
i2c-10 to i2c-12

Signed-off-by: Mohsen Dolaty <mohsen.dolaty@amd.com>
---
 arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts b/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
index 403baf16a961..2f727d74eedf 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
@@ -137,7 +137,7 @@ &i2c0 {
 };

 &i2c1 {
-	//Net name i2c10 (SCM_I2C10) - quartz side i2c12
+	//Net name i2c12 (SCM_I2C12) - LCD at 50KHz
 	// Connected to TCA9546A and to LCD HDR
 	status = "okay";

@@ -519,6 +519,7 @@ &i2c15 {
 	status = "okay";
 };

+#ifdef OPENBMC_I3C_ENABLE
 &i3c0 {
 	// P0 DIMM (A-F) SPD Access
 	status = "okay";
@@ -668,7 +669,7 @@ spd5118_3_5: spd@55 {
 		reg = <0x55 0x0 0x00000000>;
 	};
 };
-
+#endif  // OPENBMC_I3C_ENABLE
 &i2c2 {
 	// P0 APML
 	status = "okay";
--
2.25.1

